adapters/repos/db/crud.go

```diff
@@ -125,6 +124,28 @@ func (d *DB) ObjectByID(ctx context.Context, id strfmt.UUID,
 	return d.enrichRefsForSingle(ctx, result, props, additional)
 }
 
+// Object gets object with id from index of specified class.
+func (d *DB) Object(ctx context.Context, className string,
+	id strfmt.UUID, props search.SelectProperties,
+	adds additional.Properties) (r *search.Result, err error) {
```

Comment:
I would remove named return values and declare `*search.Result` and `error` variables inside method, to be consistent with other method declarations and also to not to decrease method's visibility. I'm always cautious when using named return values as they increase the risk of shadowing.

Reply:
I see your point but It is a common go practice

Reply:
Yeah, it's a pattern in go that can be used, but I'm not in favor of it, bc in my opinion it shadows method's logic. Actually I agree with the points made in this [article](https://dave.cheney.net/practical-go/presentations/gophercon-israel.html#_avoid_named_return_values)

Reply:
l agree in general with the points made in the article. However for me it's not a rule. I would use it when it makes sense to me except if we agree to make it a rule. Moreover I won't spend more time on this since this is a matter of style on which usually people don't agree 😉